lib/module.js

```diff
@@ -164,24 +164,30 @@ export default function SentryModule (moduleOptions) {
 
   // Enable publishing of sourcemaps
   if (!options.disabled) {
-    this.extendBuild((config, { isClient, isModern, isDev }) => {
-      if (!options.publishRelease || isDev) {
+    this.nuxt.hook('webpack:config', (webpackConfigs) => {
+      if (!options.publishRelease || this.options.dev) {
         return
       }
 
-      config.devtool = options.sourceMapStyle
+      webpackConfigs.forEach((config) => {
+        config.devtool = options.sourceMapStyle
```

Comment:
I would suggest to move this line after the exit conditions (at the end) as there is no need to compute the source map if they won't be used...

Reply:
What do you think @rchl ?

Reply:
I think it's needed here since we need to generate source maps for client configs so that when building server we publish with source maps (also for the client configs).